gloss/crt0.S

```diff
@@ -188,11 +188,22 @@ _skip_init:
 1:
 
   /* Check RISC-V isa and enable FS bits if Floating Point architecture. */
-  li   a4, 0x10028
+  li   a4, 0x10028 /* F, D, Q */
+  and  a4, a5, a4
+  beqz a4, 1f
+  csrr a4, mstatus
+  lui  a4, 0x2 /* set METAL_MSTATUS_FS_INIT */
+  or   a4, a5, a4
+  csrw mstatus, a4
+  csrwi fcsr, 0
+1:
+
+  /* Check RISC-V isa and enalbe VF bits if Vector extension architecture */
```

Comment:
Thanks for fixing it.

I think this is not urgent for formal release, but it is necessary for internal development.
If there is any side effect for libmetal release, it is fine to keep it in this branch, and request RVV developers to use this branch.

Reply:
I don't believe there will be any side effect with the changes. My immediate concern is in master, it could get release in the next SW release cycle, and therefore potential complicate backward compatibility when we do release vector support. I however understand the burden it will put on vector developer, but hopefully it would not be too much.

Reply:
I can wait till there is a need to ship libmetal with RVV enabled.